.gitignore

```diff
@@ -34,6 +34,7 @@ _build
 
 # This file results from running `pip -e install .` in a local repository
 *.egg-info
+.eggs
```

Comment:
It's probably worth adding this to the template of the cookiecutter if it isn't there already.